src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -360,9 +359,6 @@ def fake_quantize(
     levels: int,
     auto_broadcast: str = "NUMPY",
     name: Optional[str] = None,
-    *,
-    prefix: Optional[str] = None,
-    suffix: Optional[str] = None,
```

Comment:
As prefix/suffix have been already exposed within the 2024.0 removal of this mechanism is actually not backward compatible. Maybe the old idea can be removed separately or work simultaneously just for this fake_quantize case.

Reply:
@mitruska it's hard to make them coexists as both wrappers (either namable_op or affix one) already create Constant node which results in only one of them being used -- as already established nodes are not getting name update.

Can this removal become exception? As it is already creating not backward compatible change.

Reply:
I believe prefix/sufifx was not widely adapted feature, but please double check with the reporter that it's not currently used by NNCF.
I don't want to block it, but clarify the impact. 

cc: @KodiaqQ

Reply:
Well, from the NNCF side, there are no blockers, since we've used another approach with the constants for FakeQuantize layers. You can observe the current approach here - [model_transformer.py#L313-L326](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/openvino/graph/model_transformer.py#L313-L326).
The main wish here is to avoid name collisions in case of the automatic **Constant** layers creation.